project/Dependencies.scala

```diff
@@ -1,16 +1,14 @@
 import sbt._
 
 object Version {
-  val scalaVer        = "2.12.3"
-  val scalaParsers    = "1.0.4"
-  val scalaTest       = "3.0.1"
-  val scopt           = "3.7.0"
-  val sbtio           = "1.0.0"
-  val typesafeConfig  = "1.3.1"
+  val scalaVer        = "2.13.2"
```

Comment:
Where does this version apply? Most in here is run within sbt which is Scala 2.12 only.

Reply:
This applies to the code run in the ≠ applications (studentify, lineariteit, ...)